site/reference/schema/crd-status-convention/README.md

```diff
@@ -0,0 +1,63 @@
+# CRD Status Convention
+
+The custom controller should use the following conditions to signal whether a
+resource has been fully reconciled, and whether it has encountered any problems:
+
+**Reconciling**: Indicates that the resource does not yet match its spec. i.e.
+the desired state as expressed in the resource spec object has not been fully
+realized in the cluster. A value of True means the controller is in the process
+of reconciling the resource while a value of False means there are no work left
+for the controller.
+
+**Stalled**: Indicates that the controller is not able to make the expected
+progress towards reconciling the resource. The cause of this status can be
+either that the controller observes an actual problem (like a pod not being able
+to start), or that something is taking longer than expected (similar to the
+`progressDeadlineSeconds` timeout on Deployments). If this condition is True, it
+should be interpreted that something might be wrong. It does not mean that the
+resource will never be reconciled. Most process in Kubernetes retry forever, so
+this should not be considered a terminal state.
+
+These conditions adhere to the [Kubernetes design principles] which include
+expressing conditions using abnormal-true polarity. There is currently a
+[proposal] to change to guidance for conditions. If this is accepted, the
+recommended conditions for kpt might also change, but we will continue to
```

Comment:
This PR has merged, so we can probably remove the reference to abnormal-true polarity here. We should also mention that kpt does understand the `Ready` condition if it is found on a resource. It essentially works the same as the `Reconciling` condition, just opposite polarity.